app/src/main/java/com/limelight/binding/input/ControllerHandler.java

```diff
@@ -651,10 +651,14 @@ else if (devName.equals("Xbox Wireless Controller")) {
                     context.isNonStandardXboxBtController = true;
                 }
             }
+            else if (devName.toLowerCase().startsWith("8bitdo")) {
```

Comment:
Can you use the vendor ID rather than the name to identify 8BitDo controllers?

Reply:
Ok I'll try to find the correct vendor ID here.